features/environment.feature

```diff
@@ -1,9 +1,9 @@
 Feature: Git Town performs correctly depending on the environment
 
   Scenario Outline: Git Town commands run outside of a Git repository
-    Given I'm currently not in a git repository
+    Given my workspace is currently not a Git repository
     When I run `<COMMAND>`
-    Then I get the error "This is not a Git repository"
+    Then Git Town prints the error "This is not a Git repository"
```

Comment:
Thoughts on `Git Town` -> `it`. I don't think having the name of the tool is needed

Reply:
This is how I would describe the functionality in my own words to another person ("Git Town prints meaningful error messages"). 
To me, `it` by itself doesn't make too much sense. I like using it to refer to the subject in the previous step. We could abbreviate `Git Town` into `GT` if that helps...

Reply:
`Git Town prints meaningful error messages` is how we would describe the feature. But in the flow of Gherkin I don't think it makes sense. 

* When I run `git town sync`, then Git Town prints the error ...

That sounds weird to be and it mentions Git Town twice when it doesn't need to. Thus I prefer:

* When I run `git town sync`, then it prints the error  ...

Reply:
Okay, I like that approach, implemented. Please note that in this particular case I left it as it is, because the sentence reads `When I run <COMMAND> then Git Town prints the error ...`. In almost all other cases I have renamed it to `then it prints ...` though.